models/task.go

```diff
@@ -22,6 +26,24 @@ type TaskLinks struct {
 	Job  string `json:"job"`
 }
 
+// TaskName - iota enum of possible task names
+type TaskName int
+
+// ParseTaskName returns a TaskName from its string representation
+func ParseTaskName(taskNameStr string) (TaskName, error) {
+	cfg, err := config.Get()
```

Comment:
Never reinstantiate the config while the app is processing (this should only be done once at start up and the values passed to where they are needed in the app).

The reason is that, someone can make changes to environment variables on disk while this app is running. In this case your code will bring them in when this function gets called instead of using the configuration defined on startup.

This would put the app in a weird state where some functions it has behave in the expected way due to configurations inherited from start up while here it may behave differently due to picking up new values retrieved at this point in time.